src/win/util.c

```diff
@@ -1158,6 +1157,7 @@ int uv_getrusage(uv_rusage_t *uv_rusage) {
 
 
 int uv_os_homedir(char* buffer, size_t* size) {
+  HANDLE hToken;
```

Comment:
http://en.wikipedia.org/wiki/Hungarian_notation :-)

I think Bert is suggesting you should rename it to just `token` or `token_handle`.